packages/react/src/components/navigation/IonTabs.tsx

```diff
@@ -90,9 +84,32 @@ export const IonTabs = /*@__PURE__*/ (() =>
       }
     }
 
+    renderTabsInner(children: React.ReactNode, outlet: React.ReactElement<{}> | undefined) {
+      return (
+        <IonTabsInner {...this.props}>
+          {React.Children.map(children, (child: React.ReactNode) => {
+            if (React.isValidElement(child)) {
+              const isRouterOutlet =
+                child.type === IonRouterOutlet ||
+                (child.type as any).isRouterOutlet ||
```

Comment:
Just an observation, but maybe in these cases is better to use `unknown` than `any`

Reply:
I'll keep that in mind. I recall that we have a ticket to improve the types for this so we can make the update then. Thank you!